src/cms/utils/pdf_utils.py

```diff
@@ -0,0 +1,142 @@
+import hashlib
+import logging
+
+from xhtml2pdf import pisa
+
+from django.contrib.staticfiles import finders
+from django.core.cache import cache
+from django.db.models import Min
+from django.http import HttpResponse
+from django.template.loader import get_template
+from django.utils.translation import ugettext as _
+
+from backend.settings import STATIC_URL, MEDIA_URL
+from ..constants import text_directions
+from ..models import Language
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=too-many-locals
+def generate_pdf(region_slug, language_code, region, pages):
```

Comment:
You don't need both the `region_slug` and the `region` parameter, one is enough.

Reply:
That's true, i just changed it.